app/models/email.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Email < ApplicationRecord
+  class Association < ApplicationRecord
+    belongs_to :email
+    belongs_to :object, polymorphic: true
+  end
```

Comment:
Should it be in its own file? And maybe called `EmailAssociation`?

Reply:
I see this record as a private implementation detail of `Email` logic that should not be used in the application code. As such I'd prefer to live in a single file with `Email` - our `model` folder is already unnecessary bloated.